frontend/src/index.tsx

```diff
@@ -23,14 +21,6 @@ createRoot(document.querySelector('#root') as HTMLElement).render(
             path: AppRoute.ROOT,
             element: <App />,
             children: [
-              {
-                path: AppRoute.ROOT,
-                element: (
-                  <ProtectedRoute>
-                    <Root />
-                  </ProtectedRoute>
-                ),
-              },
```

Comment:
let's return root back. I was thinking we have two pages root and home

Reply:
Done, root component is empty